rules/slack_rules/slack_app_added.yml

```diff
@@ -7,6 +7,11 @@ LogTypes:
   - Slack.AuditLogs
 Tags:
   - Slack
+  - Persistence
+  - Create Account
+Reports:
+  MITRE ATT&CK:
+    - TA0003:T1136
```

Comment:
Would `T1505` be more applicable here? `1136` also works, but:
```
Adversaries may abuse legitimate extensible development features of servers to establish persistent access to systems. Enterprise server applications may include features that allow developers to write and install software or scripts to extend the functionality of the main application. Adversaries may install malicious components to extend and abuse server applications. 
```

Reply:
I like this, updated